### ISSUE TITLE
There seem to be a huge number of russians ruining the game for everyone else on the EU servers

### ISSUE THREAD

**Issue by yems730** [ValveSoftware/Dota2-Gameplay#8764]
### Description

There seem to be a huge number of russians ruining the game for everyone else on the EU servers, must be a bug because they have their own servers.

**Comment by flovnes** [comment]
^ Furthermore, they don't know any language but russian, so you can't even communicate with them.
It was fine when it happened once in 4 or 5 games, and you didn't get more than 2 of them, but after 7.33
I get at least 2 now, and half of my games are completely ruined.
The reason for this may be the new rating system that gave at least +1000 mmr to everyone who played a lot of dota before the patch, but these guys will quickly go back. 
I'm queuing EU West and it's really sad to see 4 russians in my team in every second game, most of my friends don't even want to play now just because of this problem, it would be cool if this problem was solved.

**Comment by MatviyPodolsky** [comment]
Agree, I think this is the biggest bug for now in Dota. Better to play against Slardar with 99999 AS bug than with few russians in team

**Comment by AnanasSok** [comment]
I don't want potrit vam game, sorry.

**Comment by yems730** [comment]
Yes it's quite unfortunate they aren't being drafted (yet) send your prayers.

**Comment by skyleo** [comment]
I search on EU West and currently in every game there are at least 3 russians on my team that don't speak English.

It's unplayable. Communication is impossible.

**Comment by MatviyPodolsky** [comment]
> I search on EU West and currently in every game there are at least 3 russians on my team that don't speak English.
> 
> It's unplayable. Communication is impossible.

Communication is impossible even if u know russian language. they behave like apes
